app/Length.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace ConverterApp;
```

Comment:
As per our last time discussion we should implement a intermediate class `Metric` which could be used later in `MetricUnitConverter` to check if both values you are passing to convert function has metric units.